docs/Installation/Installation.rst

```diff
@@ -1,10 +1,129 @@
 .. _installation:
 
-Installation
-************
+Get started
+***********
 
-.. toctree::
-    :maxdepth: 1
+Single-click installation for users (Windows-only)
+==================================================
 
-    InstallationWithAnaconda
-    InstallationFromSource
\ No newline at end of file
+Download the latest installer from https://python-microscopy.org/downloads/. Double-click
+the installer and follow instructions.
+
+
+conda installation for users
+============================
+
+Download and install `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_.
+Then, open a command prompt and enter
+
+.. code-block:: bash
+	
+    conda create -n pyme python=2.7
```

Comment:
I don't think we should old up Zach's doc-update for this, but maybe we should revisit this week whether there's anything left to do before we make py3 the default installation. If the answer is just packaging and docs (notably launchWorkers -> PYMEClusterOfOne) I think it'd be great if we push through on that this week - I'll have a go at the launchWorkers -> PYMEClusterOfOne.

Reply:
Our target for making Py3 the default is September. Before then I'd like to a) have @csoeller try it out and b) update the analysis docs etc for `PYMEClusterOfOne`. I'd be open, however, to making sure that packages for py3 are built and adding an installing on py3 section / note to the installation docs.

Reply:
These new docs tell users to install Py2 and developers to install Py3. I think this, coupled with joining image.sc, should help with beta testing Py3. I think we're far along enough to expect developers to make it through any Py3 problems. There should be none on the initial install or for general usage.

Reply:
I'm a little reluctant to tell *users* to install in an environment as (at least on windows) I've run into issues running programs from shortcuts and/or associating files with programs this way. The environment way pretty much forces people to drop into an anaconda prompt and run `conda activate pyme` before running stuff on the command line. 

Installing in the base environment avoids these issues.